aquia-inc/template-vite-react
# build(deps): upgrade yarn from 4.4.0 to 4.5.0 and dedupe dependencies

## Summary

Upgraded Yarn from version 4.4.0 to 4.5.0 and deduped dependencies.

### Added

- None

### Changed

- Upgraded Yarn from `4.4.0` to `4.5.0`.

### Deprecated

- None

### Removed

- Removed redundant lockfile entries for earlier versions of dependencies that have been updated by running `yarn dedupe`

### Fixed

- None

## How to test

1. Checkout this branch: `git checkout origin/build/upgrade-yarn`
2. Run `yarn` to install dependencies using the updated Yarn version (`4.5.0`).
3. Ensure that all dependencies are correctly installed without duplication or issues.
4. Run `yarn test` and verify that all tests pass.
5. Run `yarn build` and verify that the build succeeds.
6. Run `yarn build-storybook` and ensure that the Storybook build completes successfully.
